scripts/ubuntu-bartender/multipass-provider

```diff
@@ -4,6 +4,13 @@ function build-provider-assert-ready {
     echo "error: multipass was not found in PATH" >&2
     exit 255
   fi
+  # multipass become a strict-confined snap after 1.3.
+  # this is to create a special variable to check whether it is
+  # using strict confinement or classic confinement.
+  if snap info --verbose multipass | grep strict &>/dev/null
```

Comment:
this will also match if the multipass snap's description is ever updated to have the word strict in it, I would suggest something more specific:

```suggestion
  if snap info --verbose multipass | grep -q -P "confinement:\s+strict"
```

Reply:
actually thinking about it, see my other comment I don't think this env var is necessary at all